index.html

```diff
@@ -1,12 +1,192 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Womens Conference</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+    <link rel="stylesheet" href="css/home.css" />
+    <script
+      src="https://kit.fontawesome.com/eb94992e2c.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <nav class="top-nav desktop-only">
```

Comment:
- [ ] You did a very great job with the design, But for this project, you are supposed to stick to the [Behance template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as much as possible. Please increase the font size of your top sub-menu and place some margin from the right and make sure it matches the design on the template.


### Your design

![image](https://user-images.githubusercontent.com/83803414/158949987-e4344be7-ed3d-4f26-84a1-a600e9f9243e.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/158950086-57a951a8-be12-41b4-991f-851a8e28d75d.png)